### PR TITLE
tutorials: move vweb tutorial inside its own folder

### PR DESCRIPTION
**Pull request by StunxFS**
This PR moves the vweb tutorial into its own folder.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -126,8 +126,8 @@ jobs:
         run: ./v build-vbinaries
       - name: Test v tutorials
         run: |
-          cd tutorials/code/blog
-          ../../../v .
+          cd tutorials/building_a_simple_web_blog_with_vweb/code/blog
+          ../../../../v .
```
Comment (serkonda7) [inline .github/workflows/ci.yml:130]: I think `./v tutorials/building_a_simple_web_blog_with_vweb/code/blog`
or
```
cd tutorials
../v building_a_simple_web_blog_with_vweb/code/blog
```
is somewhat cleaner

**Review by serkonda7 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (StunxFS) [reply .github/workflows/ci.yml]: thanks ^^, it's fixed now, I used the form `./v tutorials/building_a_simple_web_blog_with_vweb/code/blog`

**Review by serkonda7 — APPROVED** [review]

**Comment by medvednikov** [comment]
What for?

The idea is to have all .md tutorials right there in the tutorials folder, for better visibility.

**Comment by serkonda7** [comment]
I think it will become unorganized of we add more tutorials.

**Comment by StunxFS** [comment]
@medvednikov 
> What for?
> 
> The idea is to have all .md tutorials right there in the tutorials folder, for better visibility.

For the same reason that serkonda mentioned above:

> I think it will become unorganized of we add more tutorials.

In my opinion, the "tutorials" folder should be organized by subfolders for each example. This way the repository will be more organized :)

```
tutorials/
        building_a_simple_web_blog_with_vweb/
        building_a_project_with_vmake/
        creating_a_basic_game_with_gg/
```

**Comment by spytheman** [comment]
`https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb/building-a-simple-web-blog-with-vweb.md`
This is too long imho, and has duplication in it.
If you are splitting the tutorial into its own folder, imho you should rename the .md file to just `readme.md`,
=> url: `https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb/readme.md`

Or it can be as Alex wanted:
=> url: `https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb.md`
*and* have a folder `tutorials/building_a_simple_web_blog_with_vweb/` that contains all the code, but then
there will be both a `folder_name/`, and a `folder_name.md` in `tutorials/`, which may look weird.

**Comment by JalonSolov** [comment]
I vote for @spytheman first suggestion.  I agree with having it organized, but the .md name doesn't have to match the folder name if it is _in_ the folder.

**Comment by StunxFS** [comment]
@spytheman 
> `https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb/building-a-simple-web-blog-with-vweb.md`
> This is too long imho, and has duplication in it.
> If you are splitting the tutorial into its own folder, imho you should rename the .md file to just `readme.md`,
> => url: `https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb/readme.md`
> 
> Or it can be as Alex wanted:
> => url: `https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb.md`
> _and_ have a folder `tutorials/building_a_simple_web_blog_with_vweb/` that contains all the code, but then
> there will be both a `folder_name/`, and a `folder_name.md` in `tutorials/`, which may look weird.

Thanks for the advice ^^

I already made the changes :).

**Pull request merged**